# Study 915 — K-1 vs 1099 🧾

**Does the tax-friendly commodity wrapper give back its convenience in performance?**

Buy a commodity ETP that holds futures directly and you own a **partnership**. Every
spring it sends you a **Schedule K-1** — often late enough to force a filing extension —
and inside an IRA it can generate taxable income the account was supposed to shelter. In
exchange, its futures gains fall under **Section 1256**: taxed **60% long-term / 40%
short-term** whatever your holding period, but **marked to market every 31 December**
whether or not you sell.

**PDBC** is the same manager's answer, branded literally *"No K-1"*. It pushes the futures
into a Cayman subsidiary so it can be a normal 1940-Act fund and send you a **Form 1099**.
The trade: you lose the 60/40 rate, you receive ordinary-income distributions — and you
gain **deferral** on everything else.

The desk's question is not which structure is prettier. It is whether the convenience shows
up as a **cost on the tape**. We race **PDBC (1099)** against **DBC (K-1)** — same manager,
same index family, same commodity beta — on daily **total-return** closes,
2014-11-07 → 2026-06-30 (2,926 days), both legs **excess of cash** (BIL).

*Numbers below are the frozen headline (`docs/results.md`, Fingerprint `0e33b87210d5`); the
only live cells run a synthetic control and say so. As-of 2026-06-30.*


## 1. Two envelopes, one commodity

DBC and PDBC are run by the same house, on the same strategy family, holding essentially the same futures. They differ in the *legal envelope* around them — and therefore in the piece of paper you get in the post each spring. If the convenient envelope quietly costs you money, the two tapes should drift apart. Here is what eleven and a half years of tape says.

In [1]:
R = {'start': '2014-11-07', 'end': '2026-06-30', 'n_days': 2926, 'fp': '0e33b87210d5', 'ric_sharpe': 0.151, 'ric_cagr': 3.01, 'ric_vol': 18.1, 'ric_dd': -49.3, 'ric_t': 0.51, 'k1_sharpe': 0.159, 'k1_cagr': 3.16, 'k1_vol': 17.9, 'k1_dd': -50.6, 'k1_t': 0.53, 'diff_ann': -0.1274, 't_diff': -0.21, 'te_ann': 5.3, 'corr': 0.9566, 'beta': 0.9626, 'r2': 0.9151, 'sharpe_adv': -0.0081, 'ci_lo': -1.0323, 'ci_hi': 0.7768, 'ci_pneg': 0.618, 'adv_ci_lo': -0.0587, 'adv_ci_hi': 0.0421, 'er_k1': 0.87, 'er_ric': 0.59, 'fee_gap_bp': 28, 'te_monthly': 1.46, 'n_months': 140, 'acf1': -0.43, 'diff_monthly': -0.1945, 't_monthly': -0.68, 'se_iid': 1.56, 'se_hac': 0.61, 'se_boot': 0.45, 'mde_iid': 3.11, 'mde_hac': 1.22, 'mde_boot': 0.91, 'era_e_n': 1547, 'era_e_diff': 0.008, 'era_e_t': 0.01, 'era_e_te': 7.04, 'era_l_n': 1377, 'era_l_diff': -0.319, 'era_l_t': -0.73, 'era_l_te': 2.0, 'n_years': 11, 'wins': 3, 'win_rate': 27, 'win_lo': 10, 'win_hi': 57, 'worst_year': 2025, 'worst_diff': -2.14, 'best_year': 2015, 'best_diff': 0.74, 'cost50_diff': -0.2136, 'cost50_t': -0.35, 'cost50_drag_bp': 8.62, 'gap_min': -0.36, 'gap_max': 0.588, 'gap_top_p0': 0.588, 'gap_top_p1': 0.219, 'gap_top_p2': -0.34, 'gap_mid_p1': 0.149, 'gap_low_p1': 0.021, 'regime_gap_low': 0.241, 'regime_gap_mid': 0.38, 'regime_gap_top': 0.438, 'pretax_cagr_k1': 3.328, 'pretax_cagr_ric': 3.055, 'aftertax_k1_mid': 2.034, 'aftertax_ric_mid': 2.183, 'usci_cagr': 4.68, 'uso_cagr': -6.7, 'bno_cagr': 1.85, 'dispersion_pp': 11.38, 'syn_planted': 4.0, 'syn_recovered': 4.95, 'syn_t': -4.03, 'syn_null_mean': 0.453, 'syn_null_sd': 1.057, 'syn_null_fire': 0}

print(f"DBC  (Schedule K-1): CAGR {R['k1_cagr']:+.2f}%   "
      f"excess-of-cash Sharpe {R['k1_sharpe']:+.3f}")
print(f"PDBC (Form 1099)  : CAGR {R['ric_cagr']:+.2f}%   "
      f"excess-of-cash Sharpe {R['ric_sharpe']:+.3f}")
print()
print(f"difference (1099 - K-1) : {R['diff_ann']:+.3f}% a year    t = {R['t_diff']:+.2f}")
print(f"95% confidence interval : [{R['ci_lo']:+.2f}%, {R['ci_hi']:+.2f}%] a year")
print(f"day-to-day correlation  : {R['corr']:.3f}")

DBC  (Schedule K-1): CAGR +3.16%   excess-of-cash Sharpe +0.159
PDBC (Form 1099)  : CAGR +3.01%   excess-of-cash Sharpe +0.151

difference (1099 - K-1) : -0.127% a year    t = -0.21
95% confidence interval : [-1.03%, +0.78%] a year
day-to-day correlation  : 0.957


## 2. The answer: nothing to see

The 1099 wrapper trailed its K-1 twin by **0.13% a year** — about a tenth of one percent. That is not a finding; it is a rounding error. The *t*-statistic is **-0.21** (anything under 2 means "indistinguishable from zero"), and the honest range around it runs from **-1.03% to +0.78%** a year — comfortably straddling nothing at all.

On risk-adjusted terms the gap is even smaller: an excess-of-cash Sharpe difference of **-0.0081**, with a range of [-0.059, +0.042]. For practical purposes these are the same investment wearing different paperwork.

## 3. "But my null might just be blind"

A finding of *nothing* is only worth reading if the test could have found *something*. So we asked the sample how big a wrapper cost it would have caught. Answer: anything above roughly **0.9-1.2% a year** would have shown up clearly.

So the correct sentence is not "there is no cost" — it is **"there is no cost bigger than about 1% a year"**. Given that the entire fee difference between the two funds is 28 basis points, that is a useful bound.

> 🔬 **For the quants:** the minimum detectable difference at |*t*| = 2 is 1.22%/yr on the HAC standard error and 0.91%/yr on the 21-day block bootstrap; the naive i.i.d. figure (3.11%/yr) is three times too pessimistic because the daily difference is strongly *negatively* autocorrelated.

## 4. The 5% "tracking error" that isn't

Measured day by day, the two funds look like they wander 5.3% apart a year — which sounds alarming. Measure the same thing month by month and it collapses to **1.46%**.

The reason is mundane: the two closing prints do not clear at the same instant, and each carries its own small premium or discount to the fund's true value. They disagree on Monday and re-converge on Tuesday — the daily difference has a lag-1 correlation of **-0.43**. That is stock-exchange plumbing, not divergence a holder ever experiences.

## 5. Year by year — and the one year that looks like a story

Across the 11 complete calendar years the 1099 wrapper won **3 of 11** (27%, and a coin flip sits inside the honest range of 10-57%). Nine of the eleven annual differences are inside ±1 percentage point.

The exception is **2025**, where PDBC trailed by **2.14 pp**. PDBC is *actively* managed — it does not mechanically copy DBC's index — so it is allowed to hold a different basket, and manager drift is the natural explanation. Be clear that this is an **interpretation, not a measurement**: we did not open the holdings files, so we cannot prove it was the basket rather than the envelope. What we can say is that the gap points the other way in 2015 (+0.74 pp), which is not how a structural wrapper cost behaves.

## 6. Now the tax — and the honest bit

Pre-tax, the two are a tie. So does tax break the tie? We modelled both regimes on the realised annual returns: the K-1 leg marked to market every year at the 60/40 rate, the 1099 leg paying ordinary tax on its distributions and deferring the rest until sale.

**Every input to that model is an assumption, not a measurement** — the marginal rates, and above all how much the 1099 fund distributes each year. So we swept both. The result across the whole grid runs from **-0.36 to +0.59 percentage points a year** — and **the sign flips inside it**.

If the fund distributes little, deferral wins and the 1099 wrapper is ahead by a few tenths of a point. If it distributes twice the collateral interest — which a fund of this type can do when its subsidiary realises gains, and which we did **not** measure for PDBC — the K-1's 60/40 rate wins by about the same margin. The after-tax winner is chosen by *your assumption about the distribution*, not by the tape. We report the range and refuse to pick.

## 7. The decision you are actually making

Over the same window, choosing between commodity vehicles was worth **11.4 percentage points a year**: USCI compounded at +4.68%, USO at **-6.70%**. Choosing between the two *wrappers* was worth **0.13 points**.

The paperwork decision is roughly **90 times smaller** than the decision it is usually bundled with — and investors spend far more time on it.

## 8. Live check — the machinery isn't asleep (synthetic, not the real tape)

Before trusting a null, check that the same code finds a cost when a cost is genuinely there. The cells below build **synthetic** wrapper pairs — two funds on a shared commodity factor — one with a real drag planted in it, one with none.

*This is simulated data. None of the numbers below are DBC or PDBC.*

In [2]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
from wrapper_tax import data, strategy as st

planted, truth = data.synthetic_daily(signal_strength=1.0, seed=915)
det = st.synthetic_detect(planted, n_boot=400)
print('SYNTHETIC — planted wrapper drag  : %.2f%% a year' % (truth['planted_drag_ann']*100))
print('SYNTHETIC — the race recovered    : %.2f%% a year  (t = %+.2f)'
      % (det['estimated_drag_ann']*100, det['t_diff']))

null, _ = data.synthetic_daily(signal_strength=0.0, seed=915)
det0 = st.synthetic_detect(null, n_boot=400)
print('SYNTHETIC — identical wrappers    : %+.2f%% a year  (t = %+.2f — well inside'
      % (det0['diff_ann']*100, det0['t_diff']))
print('                                     +/-2, i.e. indistinguishable from zero)')

SYNTHETIC — planted wrapper drag  : 4.00% a year
SYNTHETIC — the race recovered    : 4.95% a year  (t = -4.03)


SYNTHETIC — identical wrappers    : -0.95% a year  (t = -0.77 — well inside
                                     +/-2, i.e. indistinguishable from zero)


## Verdict

- **Signal — None.** The 1099 wrapper costs **0.13%/yr** against its K-1 twin, with *t* = -0.21 and a range of [-1.03%, +0.78%]; null in both halves of the sample, 3/11 annual wins, and unmoved by a punitive cost sweep. The test had the power to catch anything above ~1%/yr, so this is an informative null. The after-tax model does not rescue a signal either: its gap spans [-0.36, +0.59] pp/yr and **changes sign** with the distribution assumption.
- **Tradability — Mirage.** There is no edge here to bank — the difference is tens of basis points against 1.46-5.30% of tracking error and 11.4 pp/yr of index-choice dispersion. What *is* worth knowing is the non-finding: the No-K-1 convenience appears to be **free**. That is a good reason to prefer it, and it is not alpha.